src/MacSrc/Prefs.h

```diff
@@ -68,4 +69,3 @@ extern ShockPrefs		gShockPrefs;
 void SetDefaultPrefs(void);
 OSErr LoadPrefs(ResType resID);
```

Comment:
This is awesome! Are the 'resID' variables actually used anywhere? Seems like those could be dropped now

Reply:
Yeah, those parameters are unused and can be removed. I wanted to keep the diff small for the first PR. There are a dozen references to `SavePrefs` throughout the GameSrc, although some of them are in dead code.